<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WMainWindow</class>
 <widget class="QMainWindow" name="WMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AudioBooQer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QTreeView" name="chaptersView">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::EditKeyPressed</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="rootIsDecorated">
           <bool>true</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="goButton">
          <property name="text">
           <string>Go</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="WAudioPlayer" name="playerWidget" native="true"/>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Options</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QCheckBox" name="renameCheck">
             <property name="text">
              <string>Rename processed files</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chapterNoCheck">
             <property name="text">
              <string>Insert chapter number</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>4</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>First chapter number:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="firstChapterSpin">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>4</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Digits of chapter numbers:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="numberWidthSpin">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Tools</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>4</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>SoX:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="toolSoxEdit">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="toolSoxButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>4</number>
             </property>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>LAME:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="toolLameEdit">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="toolLameButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>4</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Options:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lameOptionsEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="openDirAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="newChapterAction"/>
    <addaction name="dissolveChapterAction"/>
   </widget>
   <widget class="QMenu" name="menu_Playback">
    <property name="title">
     <string>&amp;Playback</string>
    </property>
    <addaction name="previousAction"/>
    <addaction name="stopAction"/>
    <addaction name="playAction"/>
    <addaction name="nextAction"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Playback"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="quitAction">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="openDirAction">
   <property name="text">
    <string>&amp;Open directory...</string>
   </property>
  </action>
  <action name="newChapterAction">
   <property name="text">
    <string>Create &amp;new chapter</string>
   </property>
   <property name="toolTip">
    <string>Create new chapter</string>
   </property>
  </action>
  <action name="previousAction">
   <property name="text">
    <string>Pre&amp;vious</string>
   </property>
  </action>
  <action name="stopAction">
   <property name="text">
    <string>&amp;Stop</string>
   </property>
  </action>
  <action name="playAction">
   <property name="text">
    <string>&amp;Play</string>
   </property>
  </action>
  <action name="nextAction">
   <property name="text">
    <string>Ne&amp;xt</string>
   </property>
  </action>
  <action name="dissolveChapterAction">
   <property name="text">
    <string>&amp;Dissolve last chapter</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WAudioPlayer</class>
   <extends>QWidget</extends>
   <header>waudioplayer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chaptersView</tabstop>
  <tabstop>renameCheck</tabstop>
  <tabstop>chapterNoCheck</tabstop>
  <tabstop>firstChapterSpin</tabstop>
  <tabstop>numberWidthSpin</tabstop>
  <tabstop>toolSoxEdit</tabstop>
  <tabstop>toolSoxButton</tabstop>
  <tabstop>toolLameEdit</tabstop>
  <tabstop>toolLameButton</tabstop>
  <tabstop>lameOptionsEdit</tabstop>
  <tabstop>goButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
